LedRGB.cpp

```diff
@@ -9,21 +9,15 @@ LedRGB::LedRGB(unsigned redPin, unsigned greenPin, unsigned bluePin)
 }
 
 void LedRGB::setRGB(unsigned r, unsigned g, unsigned b) {
-    r /= 2.55;
-    g /= 2.55;
-    b /= 2.55;
-    if (checkBrightness) {
-        RedLed.setBrightness(r);
-        GreenLed.setBrightness(g);
-        BlueLed.setBrightness(b);
-    }
+    setBrightness(r);
+    setBrightness(g);
+    setBrightness(b);
```

Comment:
Red led is 100%, Green is 100%, Blue is 19.6%, when overall brightness is 100%.
Red led is 50%, Green is 50%, Blue is 9.8%, when overall brightness is changed to 50%.
Is that right?